flow/data_pipeline/query.py

```diff
@@ -0,0 +1,22 @@
+"""stores all the pre-defined query strings."""
+from enum import Enum
+from flow.data_pipeline.datapipeline_test import apply_energy_one
+
+# tags for different queries
+tags = {"energy": ["POWER_DEMAND_MODEL"], "analysis": ["POWER_DEMAND_MODEL"]}
+
+# specify the function to calculate the expected result of each query
+testing_functions = {"POWER_DEMAND_MODEL": apply_energy_one}
+
+
+class QueryStrings(Enum):
+    """An enumeration of all the pre-defined query strings."""
+
+    SAMPLE = "SELECT * FROM trajectory_table WHERE partition_name=\'{partition}\' LIMIT 15;"
+    UPDATE_PARTITION = "ALTER TABLE trajectory_table ADD IF NOT EXISTS PARTITION (partition_name=\'{partition}\');"
+    POWER_DEMAND_MODEL = "SELECT id, time, 1200 * speed * " \
```

Comment:
Hmmm is there a better way of doing this? This seems fairly clunky to change.

Reply:
Specifically, you've essentially implemented a mathematical function in a SQL string. (and I think, this function is re-implemented in several different places). Any way to standardize the power demand function?